src/commands/registerCommands.ts

```diff
@@ -216,7 +216,16 @@ async function bindWithFileAssociation(documentURI: Uri, grammarURI: Uri, docume
     title: "File Association Pattern",
     value: defaultPattern,
     placeHolder: defaultPattern,
-    prompt: "Enter the pattern of the XML document(s) to be bound."
+    prompt: "Enter the pattern of the XML document(s) to be bound.",
+    validateInput: async (input: string) => {
+      var hasMatch = false;
+      try {
+        hasMatch = await commands.executeCommand(ClientCommandConstants.EXECUTE_WORKSPACE_COMMAND, ServerCommandConstants.CHECK_FILE_PATTERN, input, documentURI.toString());
+      } catch (error) {
+        console.log(`Error while validating file pattern : ${error}`);
```

Comment:
If I understand, when there is an error on language server side, it means that validate return false and wizard cannot be used. It it the expected behavior?

Reply:
Yes, that's how it should work. Any error in the language server side, or even a failure in the actual executeCommand. I can quickly try to confirm.

Reply:
Yup, `message:'Running the contributed command: 'xml.workspace.executeCommand' failed.'` if the command on the language server side throws some kind of exception or fails in some unexpected way.